src/Sarif.Driver/DriverExtensionMethods.cs

```diff
@@ -48,14 +48,6 @@ public static bool ValidateOutputOptions(this SingleFileOptionsBase options)
                     valid = false;
                 }
             }
-            else
-            {
-                if (options.OutputFilePath == null)
-                {
-                    ReportInvalidOutputOptions(options);
-                    valid = false;
-                }
-            }
```

Comment:
>} [](start = 12, length = 1)

Note to self: It makes sense to remove this if there is now a default value for `OutputFilePath` when `Inline` is `false`. Is there?

Reply:
No. Instead it is now perfectly fine not to insist on rewriting any SARIF. i.e., you can just drive the work item filing from the input file without insisting on persisting the filing details back to the input SARIF or a new side-car file.

---
In reply to: [381008622](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381008622) [](ancestors = 381008622)